File: ios/Runner/ExampleEntity.swift

class ExampleEntity
{
	var testInt: Int = 0
	var testBoolean: Bool = false
	var testFloat: Float = 0.0
	
	init(testInt: Int,
		 testBoolean: Bool,
		 testFloat: Float)
	{
		self.testInt = testInt
		self.testBoolean = testBoolean
		self.testFloat = testFloat
	}
	
	public var description: String
	{
		return "testInt: \(testInt)\ntestBoolean: \(testBoolean)\ntestFloat: \(testFloat)"
	}
}

File: android/app/src/main/kotlin/com/juceaudio/juceaudiodemo/ExampleEntity.kt
package com.juceaudio.juceaudiodemo

import androidx.annotation.Keep

@Keep
class ExampleEntity
{
	var testInt: Int = 0
	var testFloat: Float = 0F
	var testBoolean: Boolean = false
	
	
	override fun toString(): String
	{
		return """
			testInt: $testInt
			testFloat: $testFloat
			testBoolean: $testBoolean
		""".trimIndent()
	}
}